<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MolonaViz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Study"/>
    <addaction name="actionOpen_Study"/>
    <addaction name="actionSave_As_Study"/>
    <addaction name="actionEdit_Study"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionTile_Win"/>
    <addaction name="actionCascade_Win"/>
    <addaction name="actionNext_Win"/>
    <addaction name="actionPrevious_Win"/>
    <addaction name="actionClose_Win"/>
    <addaction name="actionClose_All_Win"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Guide"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_MolonaViz"/>
   </widget>
   <widget class="QMenu" name="menuPoint">
    <property name="title">
     <string>Point</string>
    </property>
    <addaction name="actionImport_Point"/>
    <addaction name="actionOpen_Point"/>
    <addaction name="actionRemove_Point"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPoint"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockPoints">
   <property name="windowTitle">
    <string>Data Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QTreeView" name="treePoints">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTraceWindow">
   <property name="windowTitle">
    <string>Application Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEditMessages">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Sensors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeSensors">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_MolonaViz">
   <property name="text">
    <string>About MolonaViz</string>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actionTile_Win">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade_Win">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionNext_Win">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious_Win">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionBayesian_Inversion">
   <property name="text">
    <string>Bayesian Inversion</string>
   </property>
  </action>
  <action name="actionDirect_Model">
   <property name="text">
    <string>Direct Model</string>
   </property>
  </action>
  <action name="actionEstimation">
   <property name="text">
    <string>Estimation</string>
   </property>
  </action>
  <action name="actionNew_Study">
   <property name="text">
    <string>New Study</string>
   </property>
  </action>
  <action name="actionEdit_Study">
   <property name="text">
    <string>Edit Study</string>
   </property>
  </action>
  <action name="actionOpen_Study">
   <property name="text">
    <string>Open Study</string>
   </property>
  </action>
  <action name="actionSave_As_Study">
   <property name="text">
    <string>Save As Study</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionClose_Win">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_All_Win">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionImport_Point">
   <property name="text">
    <string>Import Point</string>
   </property>
  </action>
  <action name="actionRemove_Point">
   <property name="text">
    <string>Remove Point</string>
   </property>
  </action>
  <action name="actionOpen_Point">
   <property name="text">
    <string>Open Point</string>
   </property>
  </action>
  <action name="actionTest_Thread">
   <property name="text">
    <string>Test Thread</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
